spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -0,0 +1,256 @@
+    desc: Header of a SSR phase biases correction message.
+    fields:
+     - time:
+            type: GPSTime
+            desc: GNSS reference time of the correction
+     - update_interval:
+            type: u8
+            units: s
+            desc: Update interval between consecutive corrections
+     - iod_ssr:
+            type: u8
+            desc: |
+                IOD of the SSR correction. A change of Issue Of Data
+                SSR is used to indicate a change in the SSR 
+                generating configuration
+     - dispersive_bias:
+            type: u8
+            desc: |
+                Indicator for the dispersive phase biases property.
+     - mw_consistency:
+            type: u8
+            desc: |
+                Consistency indicator for Melbourne-Wubbena linear combinations
+     - n_corr:
+            type: u8
+            desc: |
+                Total number of ssr phase biases corrections.
+                First nibble is the size of the sequence (n),
+                second nibble is the zero-indexed counter 
+                (ith packet of n)
+ - PhaseBiasesValues:
+     short_desc: SSR phase biases corrections for a particular satellite.
+     desc: |
+      Code biases are to be added to pseudorange.
+      The corrections are conform with typical RTCMv3 MT1059 and 1065.
+     fields:
+     - code:
+            type: u8
+            desc: Signal constellation, band and code
+            fields:
+              - 0-7:
+                  values:
+                      - 0: GPS L1CA
+                      - 1: GPS L2CM
+                      - 2: SBAS L1CA
+                      - 3: GLO L1CA
+                      - 4: GLO L2CA
+                      - 5: GPS L1P
+                      - 6: GPS L2P
+     - integer_indicator:
+            type: u8
+            desc: Indicator for integer property
+     - widelane_integer_indicator:
+            type: u8
+            desc: Indicator for two groups of Wide-Lane(s) integer property
+     - discontinuity_counter:
+            type: u8
+            desc: |
+              Signal phase discontinuity counter. 
+              Increased for every discontinuity in phase.
+     - bias:
+            type: s32
+            units: 0.1 mm
+            desc: Phase bias for specified signal
+ - PhaseBiasesPackedContent:
+     short_desc: SSR phase biases corrections for a particular satellite.
+     desc: |
+      Code biases are to be added to pseudorange.
+      The corrections are conform with typical RTCMv3 MT1059 and 1065.
+     fields:
+     - sid:
+            type: GnssSignal
+            desc: GNSS signal identifier (16 bit)
+     - yaw:
+            type: u16
+            units: 1 / 256 semi-circle
+            desc: Satellite yaw angle
+     - yaw_rate:
+            type: s16
+            units: 1 / 8192 semi-circle / s
+            desc: Satellite yaw angle rate
+     - biases:
+            type: array
+            fill: PhaseBiasesValues
```

Comment:
I don't think this will work. I believe fill is greedy - it will take as many `PhaseBiasesValues` as there exist left in the frame. The issue is that this structure `PhaseBiasesPackedContent` is also a fill in `MSG_SSR_PHASE_BIASES` - hence the `PhaseBiasesValues` fill will override the `PhaseBiasesPackedContent`. I think generally you can only have a single fill in a message, and it has to be the last thing in the message.